tests/pytest/tools/taosdumpTest2.py

```diff
@@ -91,6 +100,30 @@ def run(self):
         tdSql.query("select count(*) from t1")
         tdSql.checkData(0, 0, self.numberOfRecords)
 
+        tdSql.execute("create database test")
+        tdSql.execute("use test")
+        tdSql.execute("create table stb(ts timestamp, c1 binary(16000), c2 binary(16000), c3 binary(10000)) tags(t1 nchar(256))")
+        tdSql.execute("insert into t1 using stb tags('t1') values(now, '%s', '%s', '%s')" % (self.generateString(16000), self.generateString(16000), self.generateString(10000)))
+        
+        os.system("rm /tmp/*.sql")
+        os.system("rm /tmp/*.avro*")
+        os.system("%staosdump -D test -o /tmp" % binPath)
+
+        tdSql.execute("drop database test")
+        tdSql.query("show databases")
+        tdSql.checkRows(0)
+
+        os.system("%staosdump -i /tmp" % binPath)
+
+        tdSql.execute("use test")
+        tdSql.error("show vnodes '' ")
+        tdSql.query("show stables")
+        tdSql.checkRows(1)
```

Comment:
it would be better if we check the value with the old one to avoid it being trimmed.

Reply:
the value is also checked in line 122